cpp/src/parquet/arrow/arrow-reader-writer-test.cc

```diff
@@ -2034,16 +2037,16 @@ struct GenerateArrayFunctor {
   double pct_null;
 };
 
-typedef std::function<void(int, std::shared_ptr<::DataType>*, std::shared_ptr<Array>*)>
+typedef std::function<void(int, std::shared_ptr<DataType>*, std::shared_ptr<Array>*)>
```

Comment:
Is this a copy from line 2015?  I think this could be written as a `using` alias which I find easier to read.